src/ripple/app/tx/impl/AMMVote.cpp

```diff
@@ -115,7 +115,7 @@ applyVote(
                 << "AMMVote::applyVote, account " << account << " is not LP";
             continue;
         }
-        auto feeVal = entry[sfTradingFee];
+        auto feeVal = entry[~sfTradingFee].value_or(0);
```

Comment:
This can safely be changed back to the original code `auto feeVal = entry[sfTradingFee];`. If the field isn't set, it'll return 0.